utils/build-presets.ini

```diff
@@ -928,6 +936,7 @@ reconfigure
 mixin-preset=buildbot_linux_crosscompile_android,tools=RA,stdlib=RD,build
 
 android-arch=aarch64
+cross-compile-hosts=android-aarch64
```

Comment:
This will need to be added for armv7 above too, but those flags are added to this one. I think I'll need to add another Android preset to keep that flag separate.

Reply:
The preset `buildbot_linux_crosscompile_android,tools=RA,stdlib=RD,build,aarch64` inherits everything from `buildbot_linux_crosscompile_android,tools=RA,stdlib=RD,build`, but as far as I remember, it also overrides those values (that's how `android-arch=` in the line above works).

You should be able to add `cross-compile-hosts=android-armv7` in the preset above, and override here with this line.

Reply:
I wasn't sure if it would override or get confused, I will try it and update this pull.

Reply:
It turns out it appends and builds them _both_, [as the help string points out](https://github.com/apple/swift/blob/2d04ed0edb8805c37308409587da26c204eadeb6/utils/build_swift/build_swift/driver_arguments.py#L555), and I confirmed by trying it. Therefore, I've split out the presets as I mentioned.

Let me know if you'd like me to rename the armv7 preset above, or you want to keep the same preset name that doesn't mention that it is for armv7.